src/o365/teams/commands/report/teams-report-useractivityuserdetail.ts

```diff
@@ -108,6 +176,9 @@ class TeamsReportUserActivityUserDetailCommand extends GraphCommand {
       Gets details about Microsoft Teams user activity by user 
       for July 13, 2019
       ${commands.TEAMS_REPORT_USERACTIVITYUSERDETAIL} --date 2019-07-13
+
+      Gets details about Microsoft Teams user activity by user for the last week and exports the report data in the specified path in csv format
```

Comment:
Seems like you've build functionality for `json` export, but your example is with `csv`. Could you please align?

Reply:
I have fixed this in the latest commit.